docs/source/getting_started/index.rst

```diff
@@ -21,9 +21,8 @@ Installation
 .. include:: install.rst
 
 
-.. toctree::
-   :hidden:
-   :maxdepth: 2
+.. autosummary::
+   :toctree: _autosummary
```

Comment:
This should be reverted. This `toctree` contains a reference to a different page, while `autosummary` is for generation of our API documentation.